### ISSUE TITLE
Website accessibility

### ISSUE THREAD

**Issue by ricky-lv426** [Exeter-Diabetes/sde-sw-framework#5]
It is necessary to be web accessibility compliant. At the very least there should be a web accessibility statement made available on the site.

See: https://www.exeter.ac.uk/staff/web/web-accessibility-training/legislation/

**Comment by ricky-lv426** [comment]
@ricky-lv426 to draft a webaccessibilty statement, and include in a branch.
